src/components/carousel-item/daikin-carousel-item.ts

```diff
@@ -48,14 +74,39 @@ export class DaikinCarouselItem extends LitElement {
   @property({ type: Boolean, attribute: false })
   active = false;
 
+  @queryAssignedElements({ slot: "title" })
+  private readonly _titles!: readonly HTMLElement[];
+
+  @queryAssignedElements({ slot: "description" })
+  private readonly _descriptions!: readonly HTMLElement[];
+
+  @state()
+  private _hasTextContents = false;
+
+  private _handleSlotChange() {
+    this._hasTextContents = !![...this._titles, ...this._descriptions].length;
```

Comment:
nit. performance improvement.

```suggestion
    this._hasTextContents = this._titleElements.length > 0 || this._descriptionElements.length > 0;
```